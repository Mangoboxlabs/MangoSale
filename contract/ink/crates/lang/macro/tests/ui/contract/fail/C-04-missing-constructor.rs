use ink_lang as ink;

#[ink::contract]
mod noop {
    #[ink(storage)]
    pub struct Noop {}

    impl Noop {
        #[ink(message)]
        pub fn noop(&self) {}
    }
}

fn main() {}
