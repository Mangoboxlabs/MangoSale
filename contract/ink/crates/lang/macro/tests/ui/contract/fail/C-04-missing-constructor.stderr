error: missing ink! constructor
  --> $DIR/C-04-missing-constructor.rs:4:1
   |
4  | / mod noop {
5  | |     #[ink(storage)]
6  | |     pub struct Noop {}
7  | |
...  |
11 | |     }
12 | | }
   | |_^
